test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-4.xml

```diff
@@ -697,7 +697,7 @@
 	<test id="Verify whether VM successfully runs in read-only shared cache with testBadBuildID and nonfatal option specified" timeout="600" runPath=".">
 		<command>$JAVA_EXE$ -Xint -Xshareclasses:testBadBuildID,verbose,readonly,nonfatal -version</command>
 		<output type="required" caseSensitive="yes" regex="no">Continue without using it as -Xshareclasses:nonfatal is specified</output>
-		<output type="success"  caseSensitive="yes" regex="yes" javaUtilPattern="yes">(OpenJDK|Java\(TM\) SE) Runtime Environment</output>
+		<output type="success"  caseSensitive="no" regex="yes" javaUtilPattern="yes">(semeru|OpenJDK|Java\(TM\) SE) Runtime Environment</output>
```

Comment:
I think this should continue to be case-sensitive, in which case the new product name is 'Semeru'.

Reply:
Changed it to `case-sensitive`, please note there are two `Semeru` occurrences, `semeru version` & `Semeru Runtime` in different cases though.
```
semeru version "1.8.0_302"
Semeru Runtime Environment (build 1.8.0_302-b07)
Eclipse OpenJ9 VM (build v0.27.0-release-5b9d76f56, JRE 1.8.0 Mac OS X amd64-64-Bit Compressed References 20210718_177 (JIT enabled, AOT enabled)
OpenJ9   - 5b9d76f56
OMR      - 9db1c870d
JCL      - 3f7b1fef7b based on jdk8u302-b07)
```

> I would favour a solution that is independent of the product or launcher name if it doesn't have to be too lax.

I think the product name change should be rare when it is finalized, so hopefully no more variants required.

Reply:
A separated PR will be proposed to match `version` & `Runtime` w/o dependence of the product/launcher names.